platform/error/platform-error-impl/src/main/java/org/codice/ddf/platform/error/handler/impl/DefaultErrorHandler.java

```diff
@@ -65,29 +59,48 @@ public void handleError(
       String uri,
       HttpServletRequest request,
       HttpServletResponse response) {
-    initIndexHtml();
+    initErrorHtml();
 
-    String stack = ExceptionUtils.getFullStackTrace(throwable);
-
-    Map<String, String> jsonMap = new HashMap<>();
-    jsonMap.put("code", String.valueOf(code));
-    jsonMap.put("message", message);
-    jsonMap.put("type", type);
-    jsonMap.put("throwable", stack);
```

Comment:
❓ Are we no longer putting the stacktrace into the HTML? I would think that could be the most valuable piece from a troubleshooting perspective.